crates/goose-cli/src/cli.rs

```diff
@@ -691,7 +691,11 @@ pub async fn cli() -> Result<()> {
                     })
                     .await;
                     setup_logging(
-                        session.session_file().file_stem().and_then(|s| s.to_str()),
```

Comment:
This is new behavior -- setup_logging takes an `Option<&str>`, but here we'd pass it `/dev/null` which becomes a log file like `20250521_154141-null.log`.

After this change, we'll pass None, which just becomes `20250521_154141.log`